src/boards/include/boards/pimoroni_pico_plus2_w_rp2350.h

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+// -----------------------------------------------------
+// NOTE: THIS HEADER IS ALSO INCLUDED BY ASSEMBLER SO
+//       SHOULD ONLY CONSIST OF PREPROCESSOR DIRECTIVES
+// -----------------------------------------------------
+
+// This header may be included by other board headers as "boards/pimoroni_pico_plus2_w_rp2350.h"
+
+// pico_cmake_set PICO_PLATFORM=rp2350
+// pico_cmake_set PICO_CYW43_SUPPORTED = 1
+
+#ifndef _BOARDS_PIMORONI_PICO_PLUS2_W_RP2350_H
+#define _BOARDS_PIMORONI_PICO_PLUS2_W_RP2350_H
+
+// For board detection
+#define PIMORONI_PICO_PLUS2_W_RP2350
+
+// --- BOARD SPECIFIC ---
+#define SPICE_SPI 0
```

Comment:
Note these are now SPCE_ in other pimoroni board headers

Reply:
(see #2048 )